### PR TITLE
Fixing problem related with public dataset tooltip when it is rerendered

### LINKED ISSUE #5179: Clicking on the public dataset icon generates a tooltip that doesn't get hidden
1. Go to the Data Library (or to the dashboard, if you imported a dataset from the library)
2. Put the cursor over the book icon.

A tooltip appears, but it won't get hidden after clicking outside or moving the mouse.

screen shot 2015-08-21 at 16 29 59

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-12-28 at 16 51 01

Fixes #5179

CR: @javierarce

### CODE REVIEW

**Comment by xavijam** [comment]
Ping! :dancer:

File: lib/assets/javascripts/cartodb/dashboard/datasets/datasets_item.js [hunk 1]
```diff
@@ -152,6 +154,18 @@ module.exports = cdb.core.View.extend({
         })
       )
     }
+
+    // Public dataset
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/dashboard/datasets/datasets_item.js]: Probably not needed, right? :)

**Comment by javierarce** [comment]
:dancer:

**Comment by xavijam** [comment]
:clap: thanks!

**Pull request merged**